### PR TITLE
added flutter and R cheat sheet

### PR DESCRIPTION
**Pull request by mirajus-salehin**
As flutter is growing in popularity, so I thought it's good to have a cheatsheet of flutter. The cheatsheet was difficult to write as JSON, so I have made it with Github flavored markdown.

What's inside the cheatsheet?

- Basics of flutter
-  Many useful codes like slider, radio button, loading indicator
- Special cheat for creating a responsive header

### CODE REVIEW

**Review by shaonkabir8 — APPROVED** [review]
অসাধারণ চিটশিটের জন্য আপনাকে ধন্যবাদ। :heart:

**Comment by mirajus-salehin** [comment]
- new cheatsheet added for R
- cheatsheet for flutter cannot be formatted as instructed because JSON doesn't support multiline strings. I've tried to make the code object a minified string but it's a mess and will be hard to maintain.

File: data/r.json [hunk 1]
```diff
@@ -0,0 +1,296 @@
+{
+    "id": "codendesc",
+    "title": "বেসিক R চিটশিট",
```
Comment (shaonkabir8) [inline data/r.json:3]: 'বেসিক R চিটশিট' এর পরিবর্তে 'R প্রোগ্রামিং' দিতে পারেন।

Comment (shaonkabir8) [inline data/r.json:3]: 'বেসিক R চিটশিট' এর পরিবর্তে 'R প্রোগ্রামিং' দিতে পারেন।

**Review by shaonkabir8 — CHANGES_REQUESTED** [review]

File: data/r.json (hunk 1 above)
Comment (mirajus-salehin) [reply data/r.json]: সাজেশনের জন্য ধন্যবাদ। :heart:

**Review by mirajus-salehin — COMMENTED** [review]
```title``` object changed as requested

**Review by sabbirshawon — APPROVED** [review]

**Pull request merged**